packages/connector-local/src/connector.ts

```diff
@@ -309,10 +312,14 @@ export default class LocalConnector implements IConnector {
     /* istanbul ignore next */
     private async onParseHTML(event: HTMLParse) {
         this._document = event.document;
+        this._base = getBaseUrl(this._document, event.resource);
         this._evaluate = this.createJsdom(event.html).window.eval;
 
         await traverse(this._document, this.engine, event.resource);
 
+        // Restoring base after traverse to avoid issues if there is more than one HTML file.
+        this._base = null;
```

Comment:
Isn't this unnecessary given we always assign `this._base` at the beginning of `onParseHTML`? If we had some sort of out-of-order issue we'd still have a bug if both HTML files had a `<base>` since `null` wouldn't be the right answer either.

Reply:
That was necessary with the old code. Removing!